test/unit/tasks/cms/cms_test.rb

```diff
@@ -5,7 +5,7 @@ class CmsTest < ActiveSupport::TestCase
 
     def setup
       @page = FactoryBot.build(:cms_partial, system_name: 'shared/pagination',
-        published: '<p>Title</p><a href="{{ part.link }}">Supertramp</a>')
+                               published: '<p>Title</p><a href="{{ part.link }}">Supertramp</a>')
```

Comment:
maybe revert this indentation only change?

Reply:
It's not directly related to the PR but it's a Rubocop fix, I think it's OK.